quick-start-with-tidb.md

```diff
@@ -332,14 +432,17 @@ Other requirements for the target machine:
         tiup cluster display <cluster-name>
         ```
 
+</div>
+
+</SimpleTab>
+
 ## What's next
 
 - If you have just deployed a TiDB cluster for the local test environment:
 
     - Learn [Basic SQL operations in TiDB](/basic-sql-operations.md)
     - [Migrate data to TiDB](/migration-overview.md)
     - Learn [TiDB key features and scenarios](/overview.md)
-    - Learn [TiDB's architecture](/tidb-architecture.md)
```

Comment:
Why this item is removed?

Reply:
The architecture is for reference. It's not as important as others. @TomShawn 

And I deleted /overview.md and /mysql-compatibility.md because they are not "what's next". According to the left-hand navigation structure, they are read before the quick start page.